packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx

```diff
@@ -157,6 +197,9 @@ const PostsPagePostHeader = ({post, classes}: {
             <Components.GroupLinks document={post} noMargin={true} />
           </div>}
           <a className={classes.commentsLink} href={"#comments"}>{ postGetCommentCountStr(post)}</a>
+          {podcastEpisode && <LWTooltip title={'Listen to this post'} className={classes.podcast}>
+              <a href="#" onClick={() => setShowEmbeddedPlayer(!showEmbeddedPlayer)}><HeadsetIcon/></a>
```

Comment:
You should have a`captureEvent` in the onClick handler, including tracking open vs close

Reply:
AnalyticsContext wouldn't work because it's not navigating?